pkg/skaffold/schema/validation/validation.go

```diff
@@ -581,3 +591,51 @@ func wrapWithContext(config *parser.SkaffoldConfigEntry, errs ...error) []error
 	}
 	return errs
 }
+
+// validateKubectlManifests
+// - validates that kubectl manifest files specified in the skaffold config exist
+func validateKubectlManifests(configs parser.SkaffoldConfigSet) (errs []error) {
+	for _, c := range configs {
+		if c.IsRemote {
+			continue
+		}
+		if c.Deploy.KubectlDeploy == nil {
+			continue
+		}
+		// validate that manifest files referenced in config exist
+		for _, pattern := range c.Deploy.KubectlDeploy.Manifests {
+			if util.IsURL(pattern) {
+				continue
+			}
+			// handle absolute manifest paths
+			base := ""
+			var err error
+			if !filepath.IsAbs(pattern) {
```

Comment:
I don't think `filepath.IsAbs(pattern) == false` will ever arise. See suggestion in `runner.go`

Reply:
I'm not sure I understand, if a user has an absolute path to a manifest - eg:
```
deploy:
  kubectl:
    manifests:
      - /tmp/deployment.yaml
```

Will this case never happen?  Is this accounted for elsewhere?  This check was because I didn't want to prepend a work-dir to an absolute path initially

Reply:
yes, if we only run this validation for selected commands, then we always convert all the `kubectl` and other deployer file paths to absolute values here:
https://github.com/GoogleContainerTools/skaffold/blob/5926b5a1d035c601c8bda70998cacbbe4677c690/pkg/skaffold/parser/config.go#L178

So you don't need this check again, and the files should be resolvable.

Reply:
I see thanks! Removed